tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -795,4 +795,27 @@ public function testCreateCustomExecution()
         $this->assertEquals($executions['body']['executions'][0]['trigger'], 'http');
         $this->assertStringContainsString('foobar', $executions['body']['executions'][0]['stdout']);
     }
+
+    public function testGetRuntimes()
+    {
+
+        $runtimes = $this->client->call(Client::METHOD_GET, '/functions/runtimes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        $this->assertEquals(200, $runtimes['headers']['status-code']);
+        $this->assertEquals(12, $runtimes['body']['sum']);
```

Comment:
In CI we only have 1 runtime 👍🏻 

For it to work locally you can check if its a positive number.

Reply:
Oh Got it 👍